src/Microsoft.AspNetCore.Mvc.TagHelpers/PartialTagHelper.cs

```diff
@@ -74,6 +75,12 @@ public object Model
             }
         }
 
+        /// <summary>
+        /// Indicates if this partial view must be rendered
```

Comment:
Could you expand on this to say, when optional, executing the tag helper will no-op if the view cannot be located. otherwise it will throw stating the view could not be found.